Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/D3D12Quad.cpp

```diff
@@ -0,0 +1,832 @@
+#include "pch.h"
+#include "stdafx.h"
+#include "D3D12Quad.h"
+
+inline std::string HrToString(HRESULT hr)
+{
+    char s_str[64] = {};
+    sprintf_s(s_str, "HRESULT of 0x%08X", static_cast<UINT>(hr));
+    return std::string(s_str);
```

Comment:
If we're using C++20 for this project (noticing "stdcpp20" in the vcxproj), then we might as well just use `std::format`.

`return std::format("HRESULT of {:08X}", static_cast<UINT>(hr))`

Reply:
Done.